eq-author/src/apollo/resolvers/index.js

```diff
@@ -0,0 +1,73 @@
+import gql from "graphql-tag";
+
+export const typeDefs = gql`
+  extend type QuestionPage {
+    isNew: Boolean!
+  }
+`;
+
+const GET_NEW_PAGES_LIST = gql`
+  query newPagesList {
+    newPagesList @client
+  }
+`;
+
+const initialiseMultiplePages = (pages, cache) => {
+  const { newPagesList } = cache.readQuery({
+    query: GET_NEW_PAGES_LIST,
+  });
+
+  pages.forEach(questionPage => {
+    newPagesList.push(questionPage.id);
+  });
+
+  cache.writeData({ data: { newPagesList } });
+};
+
+const isNewQuestionPage = (questionPage, args, { cache }) => {
+  const { newPagesList } = cache.readQuery({
+    query: GET_NEW_PAGES_LIST,
+  });
+
+  return newPagesList.indexOf(questionPage.id) > -1;
+};
+
+export const resolvers = {
+  QuestionPage: {
+    isNew: isNewQuestionPage,
+  },
+
+  Mutation: {
+    createQuestionnaire: (_, input, { cache }) => {
```

Comment:
Normally `_` is used to denote an argument you are not using. What is it actually? Can we give it a name

Reply:
It was taken from Apollo docs I was following. Now changed to root variable name